lib/rucio/cli/rule.py

```diff
@@ -154,5 +154,15 @@ def update(ctx, rule_id, lifetime, locked, source_rses, activity, comment, accou
 @click.pass_context
 def list_(ctx, did, rule_id, traverse, csv, file, account, subscription):
     """List all rules impacting a given DID"""
-    args = Arguments({"did": did, "rule_id": rule_id, "traverse": traverse, "csv": csv, "file": file, "subscription": (account if account is not None else ctx.obj.client.account, subscription)})
+    args = Arguments(
+        {
+            "did": did,
+            "rule_id": rule_id,
+            "traverse": traverse,
+            "csv": csv,
+            "file": file,
+            "account": account,
+            "subscription": subscription
```

Comment:
Is it intended that this ommits the `(account if account is not None else ctx.obj.client.account`?

Reply:
> Is it intended that this ommits the `(account if account is not None else ctx.obj.client.account`?

I think that the "Remove additional logic for pulling accounts" is about that